trace/src/main/org/jetbrains/lincheck/trace/TraceRecorderTracePoints.kt

```diff
@@ -37,7 +44,7 @@ sealed class TRTracePoint(
     abstract fun toText(verbose: Boolean): String
 }
 
-class TRMethodCallTracePoint(
+internal class TRMethodCallTracePoint(
```

Comment:
These classes should not be `internal`. They are used on the plugin side to de-serialize data from trace dump back to objects.